src/travis-ci/travis-ci-create.js

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+
+const travisCIFileContents = '' +
+    'language: node_js\n' +
+    'node_js:\n' +
+    '  - \'5\'\n' +
+    '\n' +
+    'notifications:\n' +
+    '  email:\n' +
+    '    on_success: never';
+
+export default function initTravisCI(config) {
+  console.log('Creating .travis.yml...');
+
+  fs.writeFile(`${config.projectName}/.travis.yml`, travisCIFileContents, (err) => {
+    if (err) {
+      console.log(`ERROR: Failed to write .travis.yml file\n${err.toString()}`);
```

Comment:
This is minor, but I've been writing my errors differently.  We should discuss what format we want for them exactly"

Reply:
I agree. Let's talk about that today.